Description: Disables the optional Netty based connection factory until it's updated to work with Netty 4+
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.xml
+++ b/build.xml
@@ -448,6 +448,7 @@
 
     <target name="compile" depends="ivy-retrieve,clover,build-generated">
         <javac srcdir="${java.src.dir}" destdir="${build.classes}" includeantruntime="false"
+               excludes="**/Netty*"
                target="${javac.target}" source="${javac.source}" debug="on">
             <classpath refid="java.classpath"/>
             <compilerarg value="-Xlint:all"/>
